Minio/DataModel/ObjectOperationsArgs.cs

```diff
@@ -86,4 +88,29 @@ public SelectObjectContentArgs WithRequestProgress(RequestProgress requestProgre
             return this;
         }
     }
+
+    public class PresignedPutObjectArgs : ObjectArgs<PresignedPutObjectArgs>
+    {
+        internal int Expiry { get; set; }
+
+        public PresignedPutObjectArgs()
+        {
+            this.RequestMethod = Method.PUT;
+        }
+
+        internal new void Validate()
+        {
+            base.Validate();
+            if (!utils.IsValidExpiry(this.Expiry))
+            {
+                throw new InvalidExpiryRangeException("Expiry range should be between 1 and " + Constants.DefaultExpiryTime.ToString());
```

Comment:
nit: shouldn't we say "... should be between 1s ..." since the unit of time is different for the lower and upper bound for expiry?

Reply:
Message updated @krisis . Tx. PTAL.